app/services/data-loader.js

```diff
@@ -70,6 +72,19 @@ export default class DataLoaderService extends CommonDataLoaderService {
 
     return this.#loadedLearnerGroup.get(learnerGroupId);
   }
+  async loadInstructorGroup(instructorGroupId) {
+    if (!this.#loadedInstructorGroup.has(instructorGroupId)) {
+      this.#loadedInstructorGroup.set(
+        instructorGroupId,
+        this.store.findRecord('instructor-group', instructorGroupId, {
+          reload: true,
+          include: 'users',
```

Comment:
Should also include all the things to load the courses view. `offerings.session.course,ilmSessions.course, maybe others I'm not thinking of`.

Reply:
added. the users and courses associated with the given instructor groups are all the extra data points we will need here.